packages/create-modular-react-app/template/gitignore

```diff
@@ -7,6 +7,7 @@ node_modules
 
 # testing
 /coverage
+/target
```

Comment:
"target" is an odd name for this (I assume there's some historical reason for it). Can it go into /coverage?

Reply:
Historical - since it's related to  [maven](https://maven.apache.org/surefire/maven-surefire-report-plugin/)

> The Surefire Report Plugin parses the generated TEST-*.xml files under ${basedir}/target/surefire-reports